### PR TITLE
Feat 1145 add Erpa agreement buy send

### PR DESCRIPTION
**Pull request by victordelval**
## Description

Closes: regen-network/regen-registry#1145

* Add `AgreeCheckbox` and `AgreeErpaCheckbox` components to web-components
* Add AgreeErpaCheckbox to Buy and Send forms (along with the corresponding validation)
* Link to Marketplace contracts (public Notion page) and link to terms-of-service page in the website

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. Go to https://deploy-preview-1196--regen-registry.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-components/src/components/inputs/AgreeErpaCheckbox.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  sx?: SxProps<Theme>;
+}
+
+const AgreeErpaCheckbox: React.FC<Props> = ({ sx }: Props) => {
+  return (
+    <AgreeCheckbox
+      name="agreeErpa"
+      label={
+        <>
+          I agree to the <Link to={'/'}>ERPA agreement</Link>
```
Comment (mhagel) [inline web-components/src/components/inputs/AgreeErpaCheckbox.tsx:16]: Just need a real link here, I'm sure you know.

Comment (mhagel) [reply web-components/src/components/inputs/AgreeErpaCheckbox.tsx:16]: Make sure it opens in a new tab. This component may do that already.

**Review by mhagel — APPROVED** [review]
Works great. Preapproving pending a working link.

**Comment by victordelval** [comment]
@erikalogie ready to review, thanks!

File: web-components/src/components/inputs/CheckboxLabel.tsx [hunk 2]
```diff
@@ -47,7 +49,11 @@ export default function CheckboxLabel({
         label={isLabelStringOrNumber ? <Subtitle>{label}</Subtitle> : label}
       />
       {showError && (
-        <FormHelperText error={showError}>{fieldError}</FormHelperText>
+        <FormHelperText error={showError} sx={{ ml: 9 }}>
+          <Subtitle size="sm" sx={{ color: theme.palette.error.main }}>
+            {fieldError}
+          </Subtitle>
+        </FormHelperText>
```
Comment (haveanicedavid) [inline web-components/src/components/inputs/CheckboxLabel.tsx:56]: check the console for errors here - might give you a `no nesting` error or warning. if that happens, you can set the subtitlte to render `as=“span”`

**Review by haveanicedavid — APPROVED** [review]

File: web-components/src/components/inputs/CheckboxLabel.tsx (hunk 2 above)
Comment (victordelval) [reply web-components/src/components/inputs/CheckboxLabel.tsx]: Nice! thanks @haveanicedavid

**Comment by victordelval** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**